lib/grpc/stub.ex

```diff
@@ -373,14 +373,38 @@ defmodule GRPC.Stub do
       {:ok, reply} = GRPC.Stub.recv(stream)
 
       # Reply is streaming
-      {:ok, enum} = GRPC.Stub.recv(stream)
-      replies = Enum.map(enum, fn({:ok, reply}) -> reply end)
+      {:ok, ex_stream} = GRPC.Stub.recv(stream)
+      replies = Enum.map(ex_stream, fn({:ok, reply}) -> reply end)
 
   ## Options
 
     * `:timeout` - request timeout
     * `:deadline` - when the request is timeout, will override timeout
     * `:return_headers` - when true, headers will be returned.
+
+  ## Stream behavior
```

Comment:
Perhaps move that suggestion+doc from `interop` into here and just point here from there.

Reply:
Sure! I'll make that change

Reply:
Added that doc to this function and added a general comment on the Interop.Client module . Since streams are broadly used on that module. 

https://github.com/elixir-grpc/grpc/pull/272/commits/55cbc28e3a9903ecd313e7dbb3919359c9a29ca0